heggy-heggy231/week6/weekend-hw-redo/express-intro-lab-master/server.js

```diff
@@ -1,3 +1,15 @@
 // serverside js
 //  export express here
-const express = require('express');
\ No newline at end of file
+const express = require('express');
+// set var app to express() lib functions
+const app = express();
+
+// Allow CORS: we'll use this today to reduce security so we can more easily test our code in the browser.
+app.use((req, res, next) => {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
+
+// set root dir
+app.get('/', (req, res) => res.send('Hello Heggy'));
```

Comment:
This is a great sanity check. I like that you have a lot of set stops and checks in your commit history. This will help with your sanity as you deal with more and more complex code.